src/library-authoring/data/apiHooks.ts

```diff
@@ -350,10 +350,10 @@ export const useCreateLibraryCollection = (libraryId: string) => {
 };
 
 /** Get the OLX source of a library component */
-export const useXBlockOLX = (usageKey: string) => (
+export const useXBlockOLX = (usageKey: string, version?: string) => (
```

Comment:
Instead of `version?: string` maybe we can use `version?: VersionSpec` ?

https://github.com/openedx/frontend-app-authoring/blob/f86c609ff147348e15c1a51bbd64f79b4693bdcc/src/library-authoring/LibraryBlock/LibraryBlock.tsx#L7

Reply:
Updated https://github.com/openedx/frontend-app-authoring/pull/1534/commits/73ed8461ee4ad76faee9f1e106f156ac00ef3bd9